examples/main.rs

```diff
@@ -29,10 +29,10 @@ fn main() {
         "1.0".to_string(),
         "default".to_string(),
         "20160305".to_string(),
+        "rust".to_string(),
         "en-us".to_string(),
         "raspberry-pi".to_string(),
-        "arm".to_string(),
-        "rust".to_string());
+        "arm".to_string());
 
```

Comment:
I reordered the parameters because I thought it was more intuitive if all the parameters associated with the "app" were together.